File: jdroid-android-core/src/main/java/com/jdroid/android/firebase/performance/FirebasePerformanceAppContext.java
package com.jdroid.android.firebase.performance;

import com.jdroid.android.context.BuildConfigUtils;

public class FirebasePerformanceAppContext {

	public static Boolean isFirebasePerformanceEnabled() {
		return BuildConfigUtils.getBuildConfigBoolean("FIREBASE_PERFORMANCE_MONITORING_ENABLED", true);
	}

}

File: jdroid-android-core/src/main/java/com/jdroid/android/firebase/performance/FirebasePerformanceAppContext.kt
package com.jdroid.android.firebase.performance

import com.jdroid.android.context.BuildConfigUtils

object FirebasePerformanceAppContext {

    fun isFirebasePerformanceEnabled(): Boolean {
        return BuildConfigUtils.getBuildConfigBoolean("FIREBASE_PERFORMANCE_MONITORING_ENABLED", true)
    }
}
